pages/integrations/sso.md

```diff
@@ -73,6 +73,21 @@ You can configure the session duration to any timeout between 6 hours and 8,760
 
 <%= image "session_duration/configure_session_duration.png", width: 623, height: 315, alt: "Screenshot of the Buildkite SSO Session Duration Configuration" %>
 
+## SSO session IP address pinning
+
+> 📘
+> Session IP Address Pinning is an Enterprise only feature.
+
+You can configure the SSO Session IP Address Pinning to require re-authentication when user IP changes.
```

Comment:
Can you add a bit more of the why behind someone using this feature? Thinking:

```suggestion
You can require users to reauthorize with your SSO provider when their IP address changes. This helps protect against ....
```

I also noticed that this uses "re-authenticate" where the other page uses "reauthorize." Which is the better one to use?